applications/nightly_build/test_transformers.py

```diff
@@ -165,6 +165,27 @@ def test_TFBertForQuestionAnswering(self):
         onnx_model = keras2onnx.convert_keras(model, model.name)
         self.assertTrue(run_onnx_runtime(onnx_model.graph.name, onnx_model, inputs_onnx, predictions, self.model_files))
 
+    def test_TFGPT2(self):
+        if enable_full_transformer_test:
+            from transformers import GPT2Config, TFGPT2Model, TFGPT2LMHeadModel, TFGPT2DoubleHeadsModel
+            model_list = [TFGPT2Model, TFGPT2LMHeadModel, TFGPT2DoubleHeadsModel]
+        else:
+            from transformers import GPT2Config, TFGPT2Model
+            model_list = [TFGPT2Model]
+        # pretrained_weights = 'gpt2'
+        tokenizer_file = 'gpt2_gpt2.pickle'
+        tokenizer = self._get_tokenzier(tokenizer_file)
+        text, inputs, inputs_onnx = self._prepare_inputs(tokenizer)
+        config = GPT2Config()
+        for model_instance_ in model_list:
+            keras.backend.clear_session()
+            model = model_instance_(config)
+            predictions = model.predict(inputs)
```

Comment:
This prediction works for transformers 2.5.0, but not for 2.8.0 with the errors:
`>                 batch_element.shape, self.results.shape))
E         ValueError: Mismatch between expected batch size and model output batch size. Output shape = (2, 1, 12, 10, 64), expected output shape = shape (1, 1, 12, 10, 64)
`